### ISSUE TITLE
Add to Path works on one machine but not on another

### ISSUE THREAD

**Issue by DanJ210** [sass/dart-sass#322]
So I have two identical Surface Pro 4's. One for work and one for personal. I followed instructions download the zip and adding the files to a folder on C:/ drive. I created a new environment variable for Sass instead of adding it to a main one. This worked immediately on my work computer but on my personal, powershell tells me that the cmdlet is not found. I've restarted the computer and still not found. I can navigate to the folder of Sass and of course, then the 'sass' command works. Otherwise it is not found.

The installation steps are very simple. I have both computers side by side. What could I be missing? I appreciate any help.

Thanks.

**Comment by DanJ210** [comment]
Here's something that's very odd. I reset the work computer and now the path variable I had set for sass is gone. It's just not there. Yet the command still works no problem and I'm not in the same folder as the sass directory. I'm just very confused here. It works on the computer with no environment variable. Doesn't work on the one that I current see the variable.

**Comment by DanJ210** [comment]
I see what I did wrong. I was adding as user variables not system. I've never had to do this before. The weird thing is that it did work on my work computer as a user variable. Now the one I set in the work computer is gone but sass still works. I did check system variables and there is no sass path. So I'm wondering if it's going to continue to work somehow or if it will one day break because of this.

Anyways, my apologies but maybe this can help as some feedback.
